src/psyclone/tests/core/variables_access_info_test.py

```diff
@@ -36,271 +36,15 @@
 
 '''This module tests the various classes in core.access_info.'''
```

Comment:
"..in the variables_access_info module" or is it just the `VariablesAccessInfoClass`?

Reply:
The latter - done.